docs/docs/getting_started/quickstart.md

```diff
@@ -6,24 +6,52 @@ We will install VerdictDB, create a connection, and issue a simple query to Verd
 ## Install
 
 Place the following dependency in your maven pom.xml.
+```pom
+<!-- https://mvnrepository.com/artifact/org.verdictdb/verdictdb-core -->
+<dependency>
+    <groupId>org.verdictdb</groupId>
+    <artifactId>verdictdb-core</artifactId>
+    <version>0.5.0-alpha</version>
+</dependency>
+```
 
-(*please add our maven dependency: https://mvnrepository.com/artifact/org.verdictdb/verdictdb-core/0.5.0-alpha*)
-
-(*also add the pom for H2*)
 
-```
-pom
+Also, to run verdictDB on H2 database, we need to include dependency for H2.
+```pom
+<dependency>
+    <groupId>com.h2database</groupId>
+    <artifactId>h2</artifactId>
+    <version>1.4.197</version>
+    <scope>test</scope>
+</dependency>
 ```
 
 ## Insert Data
 
 
 ```java
-Connection h2 = DriverManager.getConnection();
-h2.createStatement().execute(    
-    "create table \"sales\" (" +
+Connection h2 = DriverManager.getConnection("jdbc:h2:mem:verdictdb_example_db");
+Statement stmt = h2.createStatement();
+stmt.execute(
+    "create schema \"myschema\"");
+stmt.execute(
+    "create table \"myschema\".\"sales\" (" +
     "  product   varchar(100)," +
     "  price     double)");
+List<List<Object>> contents = new ArrayList<>();
```

Comment:
Please update this example so that 1,000 rows are inserted. Just use a for-loop to generate random numbers.